docs/rst/reference/generated/factory/eradiate.scenes.atmosphere.atmosphere_factory.rst

```diff
@@ -13,8 +13,8 @@ eradiate.scenes.atmosphere.atmosphere_factory
 
       * - ``heterogeneous``
         - :class:`HeterogeneousAtmosphere`
-      * - ``heterogeneous_new``
-        - :class:`HeterogeneousNewAtmosphere`
+      * - ``heterogeneous_legacy``
+        - :class:`HeterogeneousAtmosphereLegacy`
```

Comment:
Out of curiosity: Did you edit this file manually?

Reply:
Yes, I did. Is that wrong?

Reply:
It is generated by the `make docs-rst-api` target. I'll add a notice at the top of these files to make sure that people don't bother updating them manually.